caexporter.go

```diff
@@ -34,6 +34,11 @@ var (
 		[]string{"activity"},
 		nil,
 	)
+
+	up = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
1. please have `namespace`  for the metric
2. I wouldn't name ithe metric with `up`, the preferred name is `scrape_error`, you can see other popular exporters use this

Reply:
I think `up` metric should not be used in the exporters, this metric is usually used to indicate target downtimes/issues. But anyway, as I suggested, renaming the metric makes more sense from `up` to `scrape_error`.